### PR TITLE
This week in information security: week of July 6th

### PR DESCRIPTION
**Pull request by cneill**
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: July 10th, 2015

Sorry to be so late with this. I was out last week and had a lot of catch-up to do this week. No worries if it's too late to be posted for the week - I don't want to eat into your weekend!

Thank you! Let me know if you have any edits for me.

### CODE REVIEW

File: _posts/2015-07-06-this-week-in-infosec-jul-6.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+title: "This Week in Information Security (Week of July 6th)"
+date: 2015-07-10 23:59
+comments: true
+author: Charles Neill
+published: true
+categories:
+    - Security
+---
+
+After a week off, we're back with another week in information security! We'll dive into several high-profile breaches at the Office of Personnel Management (again) and Hacking Team a malware creator that counts several governments and law enforcement agencies as customers. The Hacking Team breach uncovered a nasty Adobe Flash 0day that they had discovered and not previously reported, which has been seen in live attacks both before and after public disclosure. On a lighter note, we also have some great guides and tools for you this week.
```
Comment (catlook) [inline _posts/2015-07-06-this-week-in-infosec-jul-6.md]: ... and Hacking Team, a malware creator ...

File: _posts/2015-07-06-this-week-in-infosec-jul-6.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+# Vulnerabilities
+
+- [__Adobe Flash explot that was leaked by Hacking Team goes wild; patch now!__][flash_0day] - __PATCHED__ - As mentioned above, the disclosures following a massive data dump from Italian firm Hacking Team included a previously-undisclosed Adobe Flash exploit. This exploit has quickly found its way into malicious malware kits and the popular Metasploit framework, and has been observed in the wild. It was even [able to exploit Google Chrome][flash_0day_chrome], which is considered the hardest browser to attack in some respsects due to its unique security controls, in at least one real-world case. You can find patches for your system [here][flash_0day_patches].
```
Comment (catlook) [inline _posts/2015-07-06-this-week-in-infosec-jul-6.md]: ... ramework, and it has been observed ...

File: _posts/2015-07-06-this-week-in-infosec-jul-6.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+[zxcvbn]: https://github.com/dropbox/zxcvbn
+[entropy]: http://security.stackexchange.com/questions/92726/how-secure-is-snowdens-margaretthatcheris110sexy-password
+[zxcvbn_demo]: https://dl.dropboxusercontent.com/u/209/zxcvbn/test/index.html
+[awesome_shell]: https://github.com/alebcay/awesome-shell
+[fzf]: https://github.com/junegunn/fzf
+[marker]: https://github.com/pindexis/marker
+
+[git_fix]: https://github.com/blog/2019-how-to-undo-almost-anything-with-git
+[github_fix]: https://help.github.com/articles/remove-sensitive-data/
```
Comment (catlook) [inline _posts/2015-07-06-this-week-in-infosec-jul-6.md]: Nice!!  -- Cat

**Comment by cneill** [comment]
Thanks, Cat! Just pushed the suggested edits, and I also bumped the date for this week.

Let me know if you need anything else from me.

**Pull request merged**